xep-0280.xml

```diff
@@ -329,7 +329,7 @@
 
 </section1>
 <section1 topic='Avoiding Carbons for a single message' anchor='avoiding'>
-  <p>Some clients might want to avoid Carbons on a single message, while still keeping all of the other semantics of Carbon support. This might be useful for clients sending end-to-end encrypted messages, for example. To exclude a &MESSAGE; from being forwarded to other Carbons-enabled resources, the sending client add a &lt;private/&gt; element qualified by the namespace "urn:xmpp:carbons:2" as a child content element to the &MESSAGE; stanza.</p>
+  <p>Some clients might want to avoid Carbons on a single message, while still keeping all of the other semantics of Carbon support. This might be useful for clients sending end-to-end encrypted messages, for example. To exclude a &MESSAGE; from being forwarded to other Carbons-enabled resources, the sending client must add a &lt;private/&gt; element qualified by the namespace "urn:xmpp:carbons:2" and a &lt;no-copy/&gt; hint according to &xep0334; as child content elements to the &MESSAGE; stanza.</p>
```

Comment:
I have pondered about this as well, and came to the conclusion that a "must" better reflects the situation. This is not a strict protocol requirement on the one hand, and "MUST" might indeed require that we bump the version, which I don't like very much.